### PR TITLE
Update documentation to use new pricing terminology

### PR DESCRIPTION
**Pull request by carolstran**
### Description

Update the customer-facing product documentation (and internal linting config) to use the new pricing terminology.

✍🏼 **In scope**
- Coverage
  - tariff → data plan
  - tariff profile → coverage policy
  - main zone → 🔪 (deprecated)
- Data plan
  - rate zone/ratezone → coverage area
  - plan coverage → coverage
  - custom rate zones → custom data plans
  - inclusive volume → data allowance
  - inclusive data volume → data allowance
  - data volume → data allowance
  - inclusive volume (unpooled) → fixed allowance
  - inclusive volume (pooled) → pooled allowance (with shared data)
  - pay as you go → pay-per-use
- Trial period
  - evaluation plan → trial plan
  - evaluation package → trial SIMs
- General - Platform & Services 
  - tariff plan → data plan
- Packages - Platform & Services 
  - standard plan → starter package
  - plus plan → plus package
  - pro plan → pro package
- Package features - Platform & Services 
  - multi-inclusive volume → 🔪 (deprecated)
- Support levels 
  - standard support → starter support
  - support plans → support levels
- Billing - Platform & Services  
  - overage fee → additional usage

🙅🏼‍♀️ **Out of scope**
- Coverage
  - operators/carriers → networks
  - traffic → usage
- Data plan
  - data consumption → data usage
- Package features - Platforms & Services
  - global VPN → OpenVPN

> **Note**
> These terms were determined based on the internal [Pricing Terminology Comparison](https://emnify.atlassian.net/wiki/spaces/PM/pages/2521432209/Pricing+Terminology+Comparison) and [Pricing Terminology: Portal update](https://emnify.atlassian.net/wiki/spaces/PM/pages/2571370742/Pricing+Terminology+Portal+update) documents.

### Tips for reviewers

🔴  **Major changes** (needs comprehensive review)
- Getting started with emnify
  - `/quickstart`
- emnify Portal
  - `/portal`
  - `/portal/device-policies`
  - `/portal/org/subscription`
  - `/portal/org/billing`
  - `/portal/roles`
- How-to guides
  - `/how-tos/data-allowances`
- Glossary
  - `/glossary`

🟡 **Minor changes** (requires lower effort review)
- emnify Portal
  - `/portal/connected-devices`
  - `/portal/reports`
- Platform services
  - `/multicloud-data-streamer/stream-types`
  - `/system-events/event-types`
- Support
  - `/support`
Developer resources
  - `/graphql/graphiql-ide`
  - `/sdks/concepts`

🟢 **Smol changes** (one word, link update, removed a screenshot, etc)
- Getting started with emnify
  - `/quickstart/order-sims`
  - `/quickstart/register-sims`
  - `/quickstart/create-device`
  - `/quickstart/troubleshooting`
- emnify Portal
  - `/sso`
  - `/workspaces`
- Connectivity services
  - `/services/sim-lifecycle-management`
  - `/services/sms`
- Platform services
  - `/system-events/usage`
- Network services 
  - `/services/global-iot-network`
  - `/services/iot-cloud-communication-platform`
- How-to guides
  - `/how-tos`
  - `/how-tos/two-factor-authentication`
  - `/sso/microsoft-active-directory`
  - `/workspaces/create` 
  - `/workspaces/link`
  - `/workspaces/switch`

### Additional Context

Internal 🎟️  [PGR-266](https://emnify.atlassian.net/browse/PGR-266)

[PGR-266]: https://emnify.atlassian.net/browse/PGR-266?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/portal/sims-and-devices/device-policies.md [hunk 1]
```diff
@@ -110,35 +110,18 @@ To remove a coverage policy, click the trash bin icon next to **Details**.
 Creating multiple coverage policies lets you differentiate between device types based on their coverage needs and the associated costs.
 :::
 
-### Tariff
+### Coverage
```
Comment (jeffstahlnecker) [inline docs/portal/sims-and-devices/device-policies.md:113]: Review with Camila if she updated the policies page on this point.

File: docs/quickstart/index.md [hunk 2]
```diff
@@ -12,41 +12,41 @@ To begin developing your IoT solutions with emnify, you'll need an emnify accoun
...
     <ul>
       <li>Re-enter the displayed CAPTCHA characters.</li>
       <li>Keep in mind that CAPTCHA shows both upper and lowercase letters, so it's important to capitalize them exactly as they're shown.</li>
-      <li>Still having trouble? <a href="https://www.emnify.com/talk-to-us" target="_blank">Contact the emnify sales team</a>.</li>
+      <li>Still having trouble? <a href="https://www.emnify.com/talk-to-us">Contact the emnify sales team</a>.</li>
     </ul>
 </details>
 
-## emnify's Evaluation Plan
+## emnify's Trial plan
```
Comment (jeffstahlnecker) [inline docs/quickstart/index.md:41]:
```suggestion
## emnify's Trial
```

File: docs/quickstart/index.md [hunk 3]
```diff
@@ -12,41 +12,41 @@ To begin developing your IoT solutions with emnify, you'll need an emnify accoun
...
 
-After signing up, your account is automatically assigned to the **Evaluation Plan**.
-With the Evaluation Plan, you have 60 days to use the issued prepaid credit for testing emnify's services.
+After signing up, your account is automatically assigned to the **Trial plan**.
```
Comment (jeffstahlnecker) [inline docs/quickstart/index.md:43]:
```suggestion
After signing up, your account is automatically assigned to the **Trial**.
```

File: docs/quickstart/index.md [hunk 4]
```diff
@@ -12,41 +12,41 @@ To begin developing your IoT solutions with emnify, you'll need an emnify accoun
...
+With the Trial plan, you have 60 days to use the issued prepaid credit for testing emnify's services.
```
Comment (jeffstahlnecker) [inline docs/quickstart/index.md:44]:
```suggestion
With the Trial, you have 60 days to use the issued prepaid credit for testing emnify's services.
```

File: docs/quickstart/index.md [hunk 5]
```diff
@@ -12,41 +12,41 @@ To begin developing your IoT solutions with emnify, you'll need an emnify accoun
...
 
-Once this expires, you'll need to [choose a new plan](https://portal.emnify.com/organisation-settings/subscription#plans).
+Once this expires, you'll need to [choose a new package](https://portal.emnify.com/organisation-settings/subscription#plans).
 
 :::tip
-Information on your current balance and other details about the Evaluation plan is available in the emnify Portal.  
+Information on your current balance and other details about the Trial plan is available in the emnify Portal.
```
Comment (jeffstahlnecker) [inline docs/quickstart/index.md:49]:
```suggestion
Information on your current balance and other details about the Trial is available in the emnify Portal.  
```

File: docs/quickstart/index.md [hunk 6]
```diff
@@ -57,7 +57,7 @@ In the **Organization Settings**, select [**Subscription**](https://portal.emnif
 
 ### Supported features
 
-In addition to the standard emnify services, the Evaluation Plan supports the following features:
+In addition to the standard emnify services, the Trial plan supports the following features:
```
Comment (jeffstahlnecker) [inline docs/quickstart/index.md:60]:
```suggestion
In addition to the standard emnify services, the Trial supports the following features:
```

File: docs/quickstart/order-sims.md [hunk 7]
```diff
@@ -42,18 +42,16 @@ Here, you can specify the amount of each SIM type you want to receive.
 />
 
 :::note
-You may see the **Order Split** option depending on how many SIMs you order.
-This allows you to split orders into [batches](/glossary#sim-batch) (for example, collections of 25, 100, or 500 SIMs).
+For larger orders, you can split your order into **Batches** (for example, collections of 25, 100, or 500 SIMs).
```
Comment (jeffstahlnecker) [inline docs/quickstart/order-sims.md:45]:
```suggestion
For larger orders, you can split your order into **batches** (for example, collections of 25, 100, or 500 SIMs).
```

File: docs/quickstart/troubleshooting.md [hunk 8]
```diff
@@ -63,7 +63,7 @@ Also, verify that the network is on emnify’s [NB-IoT coverage](https://www.emn
 - After configuration changes, make sure they're correctly applied on the device (for example, with a reboot).
 - Clicking the reset connectivity icon on [**Connected Devices**](https://portal.emnify.com/connected-devices) can also reset the network state and allow your device to freshly reattach.
 - The issue may only be present with one network or in the specific location (for example, due to high interference for this network).
-You can use the Operator Blacklist to block the network and force the change to a different network.
+You can use the networks blocklist to block the network and force the change to a different network.
```
Comment (jeffstahlnecker) [inline docs/quickstart/troubleshooting.md:66]:
```suggestion
You can use the network blocklist to block the network and force the change to a different network.
```

File: docs/support.md [hunk 9]
```diff
@@ -5,19 +5,19 @@ description: Ensuring your success with emnify products and services
...
+We provide you with a choice of different levels, a globally based customer success team, and support with global roaming to ensure that you have connectivity wherever you need it.
 
-## Support plans
+## Support levels
 
-emnify offers a variety of support plans designed to ensure that your devices operate reliably in our network.
-The **Standard** support plan is included for all customers at no additional cost.
-It's the default service plan when registering through our website using the [emnify Portal](https://portal.emnify.com/).
+emnify offers a variety of support levels designed to ensure that your devices operate reliably in our network.
+**Starter Support** is included for all customers at no additional cost.
```
Comment (jeffstahlnecker) [inline docs/support.md:13]: We should hold on the package name change.

**Review by jeffstahlnecker — CHANGES_REQUESTED** [review]

File: docs/quickstart/index.md (hunk 3 above)
Comment (carolstran) [reply docs/quickstart/index.md:43]: Anything that's bold reflects exactly how it's written in the Portal, so I'll confirm this before changing.

File: docs/quickstart/order-sims.md (hunk 7 above)
Comment (carolstran) [reply docs/quickstart/order-sims.md:45]: This bolded term reflects what's written in the Portal, so I'll keep it as-is.

File: docs/support.md (hunk 9 above)
Comment (carolstran) [reply docs/support.md:13]: Ok I reverted the changes for the Support levels. If I should hold off on the general switch from **Standard Package** to **Starter Package**, let me know.

File: docs/portal/dashboard.md [hunk 10]
```diff
@@ -17,47 +17,38 @@ Any changes are reflected in these charts within a few hours.
...
+These statistics are only shown if you've [assigned data allowances](/how-tos/data-allowances).
+If you're in a Pay-per-Use model, this section of the Dashboard is empty.
 :::
 
 **Prepaid Balance**  
 The remaining balance of your prepaid credit.
 
 :::note
-This section is only shown if you're on the [Evaluation Plan](/quickstart#emnifys-evaluation-plan).
+This section is only shown if you're on the [Trial plan](/quickstart#emnifys-trial-plan).
```
[the remark this answers is not in the document]
Comment (carolstran) [reply docs/portal/dashboard.md:37]: Updated to "Trial" but still not sure what you mean by this comment... so if you give me more specifics, then I can update further.

**Comment by carolstran** [comment]
@jeffstahlnecker FYI all of the updated events are from the MNO spec, so I'll update those docs and tag you in the PR.

**Review by jeffstahlnecker — APPROVED** [review]

**Pull request merged**